src/Core_Command.php

```diff
@@ -564,28 +570,37 @@ function wp_new_blog_notification() {
 				}
 			}
 			// WP 4.9.0 - skip "Notice of Admin Email Change" email as well (https://core.trac.wordpress.org/ticket/39117).
-			add_filter( 'send_site_admin_email_change_email', function() {
-				return false;
-			} );
+			add_filter(
```

Comment:
I'm presuming you are in the WP context here as it looks like this uses a Core filter. In that case, this can probably be simplified to:
```php
add_filter( 'send_site_admin_email_change_email', '__return_false' );
```
See: https://developer.wordpress.org/reference/functions/__return_false/